lib/change_stream.js

```diff
@@ -277,44 +392,26 @@ var createChangeStreamCursor = function(self) {
   return changeStreamCursor;
 };
 
-function getResumeToken(self) {
-  return self.resumeToken || self.options.resumeAfter;
-}
-
-function getStartAtOperationTime(self) {
```

Comment:
Do we no longer want to access `startAtOperationTime`?

Reply:
these are internal helpers, not top-level API. Since we no longer actually need the helper (its baked into the new code above), this dead code was deleted.